workers/webpack.js

```diff
@@ -119,10 +121,32 @@ function webpack(opts, callback) {
   }, function (err, stdout, stderr) {
     if (err) {
       err.output = stdout + stderr;
+      //
+      // Rebuild and rerun if the error is special
+      //
+      if (err.message.includes('npm rebuild')) {
```

Comment:
@3rd-Eden We are assuming `4.x+` with warehouse. let me make sure that string includes is in `4.x` or ill switch to `indexOf`